tests/test_divide.py

```diff
@@ -0,0 +1,5 @@
+rom pythoncalculator import divide
```

Comment:
Missing the f from from 
```suggestion
from pythoncalculator import divide
```